Competition2018/src/main/java/competition/subsystems/elevator/commands/CalibrateElevatorViaStallCommand.java

```diff
@@ -36,12 +43,15 @@ public void initialize() {
 
     @Override
     public void execute() {
-        elevator.setPower(power.get());
+            currentMonitor.updateCurrent();
+            elevator.setPower(power.get());
+            
```

Comment:
If the current is above the threshold, you also need to call `calibrateHere()` on the elevator